website_sale_one_step_checkout/models/website.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    use_osc = fields.Boolean(
+        string='Use OSC',
+        default=True,)
```

Comment:
Maybe if you set this default to False and run your tests after install you could avoid collissions with other tests